packages/apps-service/src/modules/apps/schema.gql.ts

```diff
@@ -14,7 +14,7 @@ type Mutation {
 
   createAppDatabase(id: ID!, databaseName: String!): App
```

Comment:
We need to accept the `description` and `permissions` as the input in the mutation for `createAppDatabase`.

Reply:
Updated.